src/toil_scripts/lib/files.py

```diff
@@ -0,0 +1,78 @@
+from contextlib import closing
+import os
+import errno
+import tarfile
+import shutil
+
+
+# FIXME: replace with bd2k.util.files.mkdir_p
+def mkdir_p(path):
+    """
+    It is Easier to Ask for Forgiveness than Permission
+
+    :param str path: path to directory to create
+    """
+    try:
+        os.makedirs(path)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+
+def tarball_files(tar_name, work_dir='.', prefix='', fpaths=None):
```

Comment:
No — This is identical to a previous issue you brought up.  I've removed the default `=None` and moved the argument:  

`def tarball_files(tar_name, fpaths, work_dir='.', prefix=''):`